.github/dependabot.yml

```diff
@@ -11,4 +11,9 @@ updates:
     reviewers:
       - "flexpa/devs"
     schedule:
-      interval: "daily"
+        interval: "weekly"
```

Comment:
Let's remove the open requests limit and keep it to this weekly interval. Good compromise imo. We expect them one day per week.